dashboard/src/components/ConfirmDialog/__snapshots__/ConfirmDialog.test.tsx.snap

```diff
@@ -7,14 +7,22 @@ exports[`when loading is true loading spinner matches the snapshot 1`] = `
     onCloseChange={[MockFunction]}
     size="default"
   >
-    <CdsModalContent>
-      <span>
-        Loading, please wait
-      </span>
-      <LoadingWrapper
-        loaded={false}
-      />
-    </CdsModalContent>
+    <div
+      className="center"
+    >
+      <CdsModalContent>
+        <p>
+          Loading, please wait
+        </p>
+        <div
+          className="center"
+        >
+          <LoadingWrapper
```

Comment:
FYI: Clarity team has also released a beta component of this spinner... but if the modal was stable... I can't imagine how buggy will be the beta spinner one :P

Reply:
Regarding this spinner, but completely out of the scope of this PR: don't you think the loading spinner is misplaced at the very bottom of the page?

![image](https://user-images.githubusercontent.com/11535726/110822602-83522680-8291-11eb-8e0c-36f6762e906a.png)

Perhaps a similar approach as other tools do looks better, I don't know

![image](https://user-images.githubusercontent.com/11535726/110822746-aaa8f380-8291-11eb-8b7e-c1fb83e1fe1a.png)

Reply:
yep, I do agree, it be nice to use the official spinner (we used this one "temporarily" :) ). I also agree to revisit how we render the spinner in different views. For example, when loading an application is also quite messy. Let me open a quick issue for this.

Reply:
https://github.com/kubeapps/kubeapps/issues/2539